keras/backend/tensorflow_backend.py

```diff
@@ -773,18 +813,54 @@ def cast(x, dtype):
 
 
 def update(x, new_x):
+    """Update the value of 'x' to 'new_x'.
+
+    # Arguments
+        x: A Variable.
+        new_x: A tensor of same shape as 'x'.
+
+    # Returns
+        The variable 'x' updated.
+    """
     return tf.assign(x, new_x)
 
 
 def update_add(x, increment):
+    """Update the value of 'x' by adding 'increment'.
```

Comment:
Code delimiters ` would be more appropriate than string quotes here, for `x` and `increment`.